packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_map_renderer_android.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+
+/// Android map renderer
+enum AndroidMapRenderer {
+  /// Latest renderer type.
+  latestRenderer,
+
+  /// Legacy renderer type.
+  legacyRenderer,
+
+  /// Default can be used to select system default renderer type.
+  defaultRenderer,
+
+  /// Unknown renderer type
```

Comment:
Why does this exist? It's an error to use request it, and there is no code path where we return it.